src/main/java/org/folio/service/fieldname/FieldNameConfig.java

```diff
@@ -0,0 +1,59 @@
+package org.folio.service.fieldname;
+
+import org.folio.service.fieldname.loader.IdentifierTypesLoader;
+import org.folio.service.fieldname.loader.ReferenceDataLoader;
+
+import java.util.Map;
+
+public enum FieldNameConfig {
+
+  IDENTIFIERS("identifiers", "$.{recordType}[*].identifiers[?(@identifierTypeId=={id})].value", new IdentifierTypesLoader()),
+  FIXED_LENGTH_DATA_ELEMENT("metadata.createdDate", "$.{recordType}.metadata.createdDate", MetadataParametersConstants.getFixedLengthDataElement()),
```

Comment:
I am guessing this part is incorrect, why do we need languages for metadata. We need to correct this

Reply:
I will remove FIXED_LENGTH_DATA_ELEMENT field for now, but will not remove the mechanism of adding metadata... We can easily do it in the future if we will not need it

Reply:
yes we will need the metadata mechanism. We shouldn't be removing that